src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  &--4 :nth-child(-n+4) {
```

Comment:
What about other values of modifier? Would be great to add them too so you can easily switch between amount of active stars